openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -191,6 +199,141 @@ def update_account_settings(requesting_user, update, username=None):
     _send_email_change_requests_if_needed(update, user)
 
 
+def _get_and_validate_extended_profile_form(update: dict, user, field_errors: dict) -> Optional[forms.Form]:
```

Comment:
Not sure if `update` is the best name for this field, can we try something more descriptive? Also are we not typing user on purpose?
```suggestion
def _get_and_validate_extended_profile_form(updated_data: dict, user, field_errors: dict) -> Optional[forms.Form]:
```

Reply:
Thanks! Updated: https://github.com/openedx/edx-platform/pull/37119/commits/b0fa90a7a49f0011319cbea05d4db58afc0325c2